webapp/lib/view.dart

```diff
@@ -211,26 +212,42 @@ class ConfigurationViewTagListPartial {
       ..classes.add('configure-package__tags-list');
   }
 
-  void renderTagList(List<String> tags) {
+  void renderTagList(Map<String, bool> tags) {
     tagListElement.children.clear();
-    tags.forEach((tag) {
+    tags.forEach((tag, state) {
       var tagItem = new Element.li()
         ..classes.add('configure-package__tag-item')
         ..text = tag;
       tagItem.onClick.listen((event) {
         var selectedTag = (event.target as Element);
         controller.command(controller.UIAction.configurationTagSelected, new controller.ConfigurationTagData(selectedTag: selectedTag.text.trim()));
-        tagListElement.children.forEach((t) => t.classes.remove('configure-package__tag-item-active'));
-        selectedTag.classes.add('configure-package__tag-item-active');
+      });
+      tagItem.onDragEnter.listen((event) => event.preventDefault());
+      tagItem.onDragOver.listen((event) {
+        event.preventDefault();
+        (event.target as Element).classes.add('configure-package__tag-item-active');
```

Comment:
the `onDragOver` event gets triggered every few hundred milliseconds, resulting in this class being added lots of times (or at least attempted to be added lots of times). Shouldn't this class change be done `onDragEnter` instead?

Reply:
Ok. Moved the logic to `onDragEnter`